# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1318884368 -3600
#      Mon Oct 17 21:46:08 2011 +0100
# Node ID a2f8ca655ca3c02344e6a176b2a568738c0a5ae2
# Parent  d99aa455296e0ece82bddca44859d62344470dd4
Make uicontextmenu::visible property off by default.

* graphics.h.in (uicontextmenu::properties::init): Set visible to false.

diff --git a/src/graphics.h.in b/src/graphics.h.in
--- a/src/graphics.h.in
+++ b/src/graphics.h.in
@@ -4637,16 +4637,17 @@ public:
       array_property position , Matrix (1, 2, 0.0)
     END_PROPERTIES
 
   protected:
     void init (void)
       {
         position.add_constraint (dim_vector (1, 2));
         position.add_constraint (dim_vector (2, 1));
+        visible.set (octave_value (false));
       }
   };
 
 private:
   properties xproperties;
 
 public:
   uicontextmenu (const graphics_handle& mh, const graphics_handle& p)
